Tests/APITesters/ObjCAPITester/ObjCAPITester/RCPresentedOfferingContextAPI.m

```diff
@@ -0,0 +1,24 @@
+//
+//  RCPresentedOfferingContextAPI.m
+//  ObjCAPITester
+//
+//  Created by Josh Holtz on 2/14/24.
+//
+
+#import "RCPresentedOfferingContextAPI.h"
+
+@import StoreKit;
+@import RevenueCat;
+
+@implementation RCPresentOfferingContextAPI
+
++ (void)checkAPI {
+    RCPresentedOfferingContext *poc = [[RCPresentedOfferingContext alloc] initWithOfferingIdentifier:@"" placementIdentifier:nil];
+    poc = [[RCPresentedOfferingContext alloc] initWithOfferingIdentifier:@"" placementIdentifier:@""];
```

Comment:
We should have API tests without passing a `placementIdentifier`.

Reply:
Objective-C doesn't have that API 🤷‍♂️  It only has this one signature where `placementIdentifier`  nullable so this test passes a `nil` and a `@""`

Reply:
Ahh ok, Since we haven't released the other one it should be ok 👍

Reply:
I lied... I just added that convince initializer because why not 🤷‍♂️ So I'm adding it to the API tester now 😇